### PR TITLE
CD Pipeline

### PR DESCRIPTION
**Pull request by CarterMacLennan**
casual 130+ commits...... (80% of them is just triggering the pipeline)

### CODE REVIEW

File: Service/db/init-mongo.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+db.createUser(
```
Comment (nathanielcaron) [inline Service/db/init-mongo.js:1]: What is this?

Comment (CarterMacLennan) [reply Service/db/init-mongo.js:1]: ahh thx u, I will remove. I was looking for that actually but I thought I had removed it already.

**Comment by nathanielcaron** [comment]
Note: Once we merge this in, we'll have to update our local `.env` files to use port number 3001 for the service.

File: application/Dockerfile.dev [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Specify a base image
+FROM node:14.17.0-alpine as alpine
+
```
Comment (fredy20501) [inline application/Dockerfile.dev:3]: Are we doing anything with this `Dockerfile.dev` file? If not we might want to delete this file.

File: Service/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+WORKDIR /app
+
+COPY package.json .
+RUN npm install
+COPY . .
+
+EXPOSE 3001
+
+CMD ["npm","run","dev"]
```
[the remark this answers is not in the document]
Comment (nathanielcaron) [reply Service/Dockerfile:11]: Yes.
Once again, this package must be installed globally `npm install pm2 -g`.
See: https://www.npmjs.com/package/pm2

File: nginx/configs/default.conf [hunk 4]
```diff
@@ -0,0 +1,20 @@
+upstream api {
+    server api:3001;
+}
+
+upstream client {
+    server client:3000;
+}
+
+server {
+    listen 80;
```
Comment (sahilsaini432) [inline nginx/configs/default.conf:10]: Is this port 80 here for the EC2 Container?

Comment (nathanielcaron) [reply nginx/configs/default.conf:10]: That is what Nginx uses

"Port 80 is the port number assigned to commonly used internet communication protocol, Hypertext Transfer Protocol (HTTP)"

**Review by fredy20501 — APPROVED** [review]
Great work Carter! 🥇

**Review by sahilsaini432 — APPROVED** [review]
Nice Work Carter 💯

**Review by JoaoPontes98 — APPROVED** [review]

**Review by nathanielcaron — APPROVED** [review]

**Comment by nathanielcaron** [comment]
Good stuff 💯

**Pull request merged**